### PR TITLE
Reduce esm2 and geneformer test burden

### PR DESCRIPTION
**Pull request by sichu2023**

### CODE REVIEW

**Review by pstjohn — APPROVED** [review]
Thanks for shortening these! Will be interested to see the duration outputs

File: sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/train_geneformer.py [hunk 1]
```diff
@@ -541,7 +548,14 @@ def get_parser():
         default=None,
         help="Path to the checkpoint directory to restore from. Will override `--resume-if-exists` when set.",
     )
-
+    parser.add_argument("--num-layers", type=int, default=6, help="Number of layers in geneformer. Default to 6.")
```
Comment (jstjohn) [inline sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/train_geneformer.py:551]: THANK YOU!

**Review by jstjohn — APPROVED** [review]

**Comment by sichu2023** [comment]
./build-ci

**Pull request merged**